src/pages/api/preview.js

```diff
@@ -0,0 +1,23 @@
+import { axiosGet } from '../../store/axios';
+
+export default async function handler(req, res) {
+  try {
+    const previewModeCheck = await axiosGet('/',
+      {
+        headers: {
+          'X-PREVIEW-MODE': 'preview',
+          authorization: `Bearer ${req.query.token}`,
+        },
+      });
+    if (previewModeCheck.previewMode) {
+      res.setPreviewData({});
+      res.status(200).json({ message: "Preview mode turned on! The content you'll see now will include anything marked as saved but not published in storyblok.", allowed: true });
```

Comment:
Wording may need some slight adjusting… it currently sounds like published content is not shown when in preview mode – maybe something more along the lines of "You'll now see all the latest content including any new draft content that's not yet published."

Reply:
Updated